### ISSUE TITLE
SA1003 and SA1022 are reported for the same whitespace error

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#254]
In the following code, both SA1003 and SA1022 are reported for the `+` token. In this situation, SA1003 should be suppressed.

```csharp
var x = + 3;
```

**Comment by Noryoko** [comment]
Grabbing this.
